sashimi/hardware/scanning/mock.py

```diff
@@ -1,4 +1,4 @@
-from sashimi.hardware.scanning.interface import AbstractScanInterface
+from sashimi.hardware.scanning.__init__ import AbstractScanInterface
```

Comment:
hmm I don't think importing from __init__ is good practice, the reason I had the separate .py files is to prevent circular imports (if the parent module (scanning) has e.g. the dictionary, then it has to import both ni and mock, which in turn need the interface, so there would be a loop.

Reply:
So what I could come up to prevent annoying circular imports is to have all dictionaries of any hardware in the top module (sashimi.hardware), however I don't know what are the PEP guidelines/recommendations here if there are any. To me it makes sense that raw materials to build wrappers, like abstract classes, are imported directly from their respective parent modules as they reflect the very bottom of such module.

Reply:
As far as I read PEP8 there are no explicit recommendations, what I see in other packages is e.g. that there is a base.py file which contains the abstract class (which in turn inherits [AbstractBaseClass or ABC](https://docs.python.org/3.8/library/abc.html#abc.ABCMeta) ) I don't have strong opinions either way, as long as it's consistent